test/ibilinear-microkernel-tester.h

```diff
@@ -15,6 +15,8 @@
 #include <vector>
 
 #include <gtest/gtest.h>
+#include "next_prime.h"
+#include "xnnpack.h"
```

Comment:
This is duplicating the include below. Includes should now be fully qualified from the root of the repo.